store/queries/post_hotness.sql

```diff
@@ -0,0 +1,24 @@
+-- name: DeleteOldPostHotness :execrows
+DELETE FROM post_hotness
+WHERE generated_at < NOW() - sqlc.arg(retention_period)::INTERVAL;
+
+-- name: MaterializeClassicPostHotness :one
```

Comment:
A test covering this query would be awesome - I think there's a lot that could go wrong here.